index.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: black;
+}
```

Comment:
- [ ] Good work 😃 but here is one thing left need to be done. You should use [Github inline comments feature ](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) at least once like how I selected a few lines and give some suggestions to you 🍀